hieradata/dev/wso2/wso2greg/5.2.0/default/publisher.yaml

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+classes:
+  - wso2greg
+wso2::pack_filename: "%{::product_name}-%{::product_version}.zip"
+wso2::pack_extracted_dir: "%{::product_name}-%{::product_version}"
+
+wso2::template_list :
+#        - repository/conf/datasources/social-datasources.xml
+        - repository/conf/identity/sso-idp-config.xml
+
+# Host mapping to be made in etc/hosts
+wso2::hosts_mapping :
+  greg-node :
+    ip_address : 127.0.0.1
+    hostname : greg-node-1
+  localhost :
+    ip_address : 127.0.0.1
+    hostname : localhost
+
+wso2::service_name: wso2greg
+wso2::hostname : wso2greg-publisher
```

Comment:
`hostname` and `mgt_hostname` follow a standard format now, a distinct one in the `default` platform and the `kubernetes` platform.  Please refer the Hiera YAML Style Guide for WSO2 Puppet Modules [1]. 

[1] - https://docs.wso2.com/display/PM200/Hiera+YAML+Style+Guide